clang/test/CodeGenOpenCL/builtins-amdgcn-gfx950-err.cl

```diff
@@ -14,8 +14,10 @@ typedef unsigned int uint;
 typedef unsigned int uint2 __attribute__((ext_vector_type(2)));
 typedef half __attribute__((ext_vector_type(2))) half2;
 typedef short __attribute__((ext_vector_type(2))) short2;
+typedef float __attribute__((ext_vector_type(2))) float2;
 
-void test(global uint* out, global uint2* out_v2u32, uint a, uint b, global half2* out_v2f16, global float* out_f32, float scale, global short2* out_v2i16, float src0, float src1) {
+void test(global uint* out, global uint2* out_v2u32, uint a, uint b, global half2* out_v2f16, global float* out_f32, float scale, global short2* out_v2i16, float src0, float src1,
+          global float2* out_v2f32) {
```

Comment:
the format is completely off here

Reply:
I am not touching any formatting, this is what's in the branch. clang-format ignores tests